.github/workflows/tests.yml

```
@@ -1,6 +1,11 @@
 name: Tests
 
 on:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)